coalib/results/TextPosition.py

```diff
@@ -31,3 +46,69 @@ def line(self):
     @property
     def column(self):
         return self._column
+
+    @property
+    def position(self):
+        if self._position is None:
+            # _position is calculated lazily. i.e. ONLY when position
+            # is asked (for the first time) If a user never tries to use
+            # `position` it is never computed.
+            if self._text is not None:
+                self._position = self.from_line_col(
+                     self._text, self.line, self.column, create_object=False)
+            else:
+                return None
+        return self._position
+
+    @classmethod
+    def from_line_col(cls, text, line, column, create_object=True):
+        """
+        Calculates position in a file from line number and column.
+        Uses \\n as the newline character. Position starts from 0.
+
+        :param text:   A tuple of lines in which position is to be calculated.
+        :param line:   The line through which position is identified.
+        :param column: Column through which position is idnetified.
+        :return:       Position in the text, identified by line, column
```

Comment:
Here, at the end of sentence, probably should be dot. And, maybe, `line and column` would be easier to read.